aeon/distances/tests/test_utils.py

```diff
@@ -48,3 +33,42 @@ def test_reshape_pairwise_to_multiple():
     assert x2.shape == y2.shape == (5, 1, 10)
     y = np.random.rand(5)
     assert x2.shape == y2.shape == (5, 1, 10)
+
+
+def _make_3d_series(x: np.ndarray) -> np.ndarray:
+    """Check a series being passed into pairwise is 3d.
+
+    Pairwise assumes it has been passed two sets of series, if passed a single
+    series this function reshapes.
+
+    If given a 1d array the time series is reshaped to (m, 1, 1). This is so when
+    looped over x[i] = (1, m).
+
+    If given a 2d array then the time series is reshaped to (d, 1, m). The dimensions
+    are put to the start so the ts can be looped through correctly. When looped over
+    the time series x[i] = (d, m).
```

Comment:
Isn't it rather like this?
```suggestion
    If given a 1d array the time series is reshaped to (1, 1, m). This is so when
    looped over x[i] = (1, m).

    If given a 2d array then the time series is reshaped to (d, 1, m). The dimensions
    are put to the start so the ts can be looped through correctly. When looped over
    the time series x[i] = (1, m).
```

Reply:
Excellent spot yes!